examples/feature-factory/ponder.config.ts

```diff
@@ -2,8 +2,8 @@ import { createConfig } from "@ponder/core";
 import { parseAbiItem } from "abitype";
 import { http } from "viem";
 
-import { LlamaCoreAbi } from "./abis/LlamaCoreAbi";
-import { LlamaPolicyAbi } from "./abis/LlamaPolicyAbi";
+import { LlamaCoreAbi } from "./abis/LlamaCoreAbi.js";
+import { LlamaPolicyAbi } from "./abis/LlamaPolicyAbi.js";
```

Comment:
Damn, I forgot that this will be necessary if we use `"type": "module"` in the project package.json. I'm a bit concerned that the `.js` file extension is going to confuse people quite a lot.

Reply:
I think they can learn

Reply:
I take this back, I think there is a way around it